src/canvas/Geometry/FaceFinder.tsx

```diff
@@ -33,7 +34,7 @@ export class FaceFinder {
         /**
          * Starting from each edge, find the face to its left by cycling 
          * back to the same edge, always choosing the path that curves 
-         * the most counterclockwise at intersections.
+         * the most in the positive rotation direction at intersections.
```

Comment:
I tried to remove all "clockwise" or "counterclockwise" explanations because I realized they are ambiguous. In a "math" plane, with the y axis pointing up, the positive direction of rotation is counterclockwise. But in graphics, the y axis points down, and that reverses everything, and the positive direction of rotation is clockwise. Because of this ambiguity, I tried to stick with "positive rotation direction" or "negative rotation direction".